src/test/java/study/racingcar/domain/RacingGameTest.java

```diff
@@ -5,23 +5,37 @@
 import java.util.Arrays;
 import java.util.List;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 class RacingGameTest {
 
-	@Test
-	@DisplayName("레이싱게임 생성 테스트")
-	void create() {
-		//given
+	private RacingGame game;
+	private int tryNo;
+
+	@BeforeEach
+	void setUp(){
```

Comment:
```suggestion
	void setUp() {
```
코딩 컨벤션 확인해주시면 좋을 것 같아요.